src/bootstrap/config/tests.rs

```diff
@@ -159,3 +159,8 @@ fn override_toml_duplicate() {
         |&_| toml::from_str("changelog-seen = 0").unwrap(),
     );
 }
+
+#[test]
+fn profile_user_dist() {
+    assert_eq!(parse("profile = \"user\""), parse("profile = \"dist\""));
```

Comment:
this ends up calling `Config::parse_inner`, which in tests never reads anything from disk, and so the test is useless. it also doesn't test the python code.
```
; cat config.toml 
profile = "user"
changelog-seen = 2
; x c            
Traceback (most recent call last):
  File "/Users/jyn/src/rust/x.py", line 50, in <module>
    bootstrap.main()
  File "/Users/jyn/src/rust/src/bootstrap/bootstrap.py", line 1107, in main
    bootstrap(args)
  File "/Users/jyn/src/rust/src/bootstrap/bootstrap.py", line 1060, in bootstrap
    with open(include_path) as included_toml:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jyn/src/rust/src/bootstrap/defaults/config.user.toml'
```
the right place to modify is `parse_inner`, like i said in the issue
https://github.com/rust-lang/rust/blob/4d74d73569e07672b101c248a2dcc396230a3da4/src/bootstrap/config.rs#L1137

Reply:
Modified `parse_inner` and updated the test, verifying it works. I'm not sure how to test the Python side with the way I've introduced the aliases